javascript/ql/src/Security/CWE-078/examples/indirect-command-injection.js

```diff
@@ -0,0 +1,5 @@
+var cp = require("child_process");
+
+const args = process.argv.slice(2);
+const script = path.join(packageDir, 'bin', 'main.js');
```

Comment:
I know it's just an example, but perhaps we could use `__dirname` instead of `packageDir` to make it slightly more self-contained?